### ISSUE TITLE
Splitting translatable strings for merged cells makes translation more flexible

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#14581]
### Is your feature request related to a problem? Please describe.
As far as I know, Western users are more accustomed to the reporting method of row xx column xx, but for Chinese users, on the contrary, Chinese users are more accustomed to the method of xx row xx column.
However, for vertical and horizontal merges, they share the same translatable string, which is "through %s"

NVDA reports the following:
* column XX through YY
* row XX through YY

The following is the Chinese translation:
* XX 列到 YY 列
* XX 行到 YY 行

In short, we need to append 'row' or 'column' after the translatable string "through %s".

### Describe the solution you'd like
Splitting translatable strings for merged cells makes translation more flexible
### Describe alternatives you've considered
None
### Additional context
None

**Comment by cary-rowen** [comment]
Hi @CyrilleB79 
The translatable strings involved in this issue are as follows:
```
#. Translators: Speaks current row number (example output: row 3).
#, python-format
msgid "row %s"


#. Translators: Speaks the row span added to the current row number (example output: through 5).
#. Translators: Speaks the column span added to the current column number (example output: through 5).
#, python-format
msgid "through %s"

#. Translators: Speaks current column number (example output: column 3).
#, python-format
msgid "column %s"
```
